docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -98,6 +98,9 @@ The binary ("B") format specifier converts a number to a string of binary digits
 
 The precision specifier indicates the minimum number of digits desired in the resulting string. If required, the number is padded with zeros to its left to produce the number of digits given by the precision specifier.
 
+For <xref:System.Numerics.BigInteger>, positive values always have a leading zero to distinguish them from negative values. This ensures the output round-trips to the original value when parsed.
+For instance, the number `3` converted with the format specifier `"B2"` is `011` because the binary number `11` represents the negative value `-1`.
```

Comment:
It might be worth clarifying that `11` is interpreted as a signed two's complement value that is 2-bits wide. Thus `1`, `11`, `111`, `1111`, etc are all interpreted as `-1`. This allows `BigInteger` to always use the shortest roundtrippable string when formatting and to support inputs from any smaller signed type without loss of data.

This is different from `int.Parse` and similar, where:
```csharp
sbyte x = -1;
int y = int.Parse(x.ToString("B")); // 255
```

Which is due to them having a fixed number of bits and so interpreting any unspecified bits as being `0`

Reply:
@tannergooding Thank you for breaking my brain. How does this sound?

```suggestion
For instance, the number `3` converted with the format specifier `"B2"` is `011` because the binary number `11` represents the negative value `-1`. Negative numbers are interpreted as a signed two's complement value that's 2-bits wide. This allows <xref:System.Numerics.BigInteger> to always use the shortest round-trippable string when formatting, especially with smaller negative numbers.
```

Reply:
Given that the `2` here is just an example, maybe:

```suggestion
For instance, the number `3` converted with the format specifier `"B2"` is `011`. That's because the binary number `11` represents the negative value `-1`, as it is interpreted as a number with exactly `2` bits due to the `"B2"` format.
```

(I've been thinking about something of the form "it's like `intN.Parse` if you had an `intN` type", but adding one extra concept C# doesn't actually have to understand BigIntege is probably not a great idea)